.github/ISSUE_TEMPLATE/marketing-request.md

```diff
@@ -0,0 +1,53 @@
+---
+name: Marketing Request
+about: Get help getting a message out to our community through coordinated outreach no email, social media, and other public channels.
+title: 'REQUEST: New communication about <topic>'
+labels: area/contributor-comms, sig/contributor-experience
+assignees: '@jonasrosland'
```

Comment:
hm -- it may not be best to assign to one person for bus-factor. Maybe ping a github team instead? would need to create one in k/org, but everyone would be notified of the request and folks could self-assign.

Reply:
Sounds good - did we land on a team?